doc/metrics.md

```diff
@@ -15,9 +18,15 @@ Number of DataVolumes pending for default storage class to be configured. Type:
 ### kubevirt_cdi_import_pods_high_restart
 The number of CDI import pods with high restart count. Type: Gauge.
 
+### kubevirt_cdi_openstack_populator_progress_total
+Progress of volume population. Type: Counter.
+
 ### kubevirt_cdi_operator_up
 CDI operator status. Type: Gauge.
 
+### kubevirt_cdi_ovirt_progress_total
```

Comment:
@arnongilboa shouldn't the metric name include populator like it is for openstack?

Reply:
@arnongilboa you update the metrics description to be clearer? The are the same for both the openstack and ovirt metrics

Reply:
@bennyz please consider doing it in a follow-up PR

Reply:
Sure. I'm OK with that.